draft-ietf-core-senml.md

```diff
@@ -416,6 +416,15 @@ to describe that a resource is available in a SenML format in the
 first place. The relevant media type indicator is included in the
 Content-Type (ct=) attribute.
 
+## Configuration and Actuation usage
+
+While typical use of SenML is retrieving sensor data, SenML can also
```

Comment:
I'd like to delete the "While typical use of SenML is retrieving sensor data," and juse leave and not get into what is typical.